core/blockchain_test.go

```diff
@@ -1747,8 +1748,8 @@ func TestInsertReceiptChainRollback(t *testing.T) {
 // overtake the 'canon' chain until after it's passed canon by about 200 blocks.
 //
 // Details at:
-//  - https://github.com/scroll-tech/go-ethereum/issues/18977
-//  - https://github.com/scroll-tech/go-ethereum/pull/18988
+//   - https://github.com/scroll-tech/go-ethereum/issues/18977
```

Comment:
We should avoid this changes on the blanks to kept the comment style consistent.

I guess this unexpected updating is induced by golang 1.19/20 so in our project I advice to stick in 1.18

Reply:
Yes, I have try my best to resolve this comment style inconsistent problems just now, you may refresh and check